pyqtgraph/functions.py

```diff
@@ -75,9 +75,8 @@ def siScale(x, minVal=1e-25, allowUnicode=True):
             pref = SI_PREFIXES[m+8]
         else:
             pref = SI_PREFIXES_ASCII[m+8]
-    p = .001**m
-    
-    return (p, pref)    
```

Comment:
The `return` statement being removed looks like a mistake?

Reply:
To fix this, just commit the fix to your branch `alfonnews:Fix_siScale_imprecision_errors`. Thank you!